packages/order/src/models/adjustment-line.ts

```diff
@@ -27,6 +27,7 @@ export default abstract class AdjustmentLine {
   code: string | null = null
 
   @Property({ columnType: "numeric" })
+  @BigNumberField()
```

Comment:
Do we need the ambiguity in typings for amount?

Reply:
Not sure I understand.
While BigNumber is not supported everywhere I believe it makes sense to keep it as `BigNumber | number`.

Reply:
I was thinking `number` might not be necessary, but I guess it all depends on how the module is used.